src/widgets.js

```diff
@@ -99,13 +99,13 @@ define([
     };
 
     var normal = function(mug, options) {
-        var path = options.path,
+        var path = options.widgetValuePath || options.path,
```

Comment:
This makes defining virtual properties for the purpose of displaying an alternate widget very simple.